### PR TITLE
Polls - 2023-05-08

### PR DESCRIPTION
**Pull request by LongForWisdom**

### CODE REVIEW

File: governance/polls/Ratification Poll for MIP Amendment Subproposals - MIP102c2-SP7 - May 08, 2023.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+* [Canonical Proposal Version](https://github.com/makerdao/mips/blob/e22e5ed2be970e1b285553028db7eed123531803/MIP102/MIP102c2-Subproposals/MIP102c2-SP7.md)
+* [Latest Proposal Version](https://mips.makerdao.com/mips/details/MIP102c2SP7)
+* [Proposal Discussion Thread](https://forum.makerdao.com/t/mip102c2-sp7-mip-amendment-subproposals/20381)
+
+## Outcomes
+
+This poll implements a **Minimum Positive Participation** value. The Minimum Positive Participation is currently set to **10,000 MKR**.
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option AND the votes for the 'Yes' option exceed 10,000 MKR, then the following actions will be taken:**
+* The Arbitration Facilitators will confirm its passage by marking the proposal **Accepted** on the [MIPs portal](https://mips.makerdao.com/mips/list) and across all other relevant governance mediums.
```
Comment (blimpa) [inline governance/polls/Ratification Poll for MIP Amendment Subproposals - MIP102c2-SP7 - May 08, 2023.md:54]:
```suggestion
* The Arbitration Facilitators will confirm its passage by marking the proposal **Accepted** on the [MIPs portal](https://mips.makerdao.com/mips/list) and across all other relevant governance mediums. 
* The amendment pull request will be merged into the MIP repository.
```

File: governance/polls/Ratification Poll for MIP Amendment Subproposals - MIP102c2-SP7 - May 08, 2023.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+* Any further work required to implement the proposal will be tasked to the relevant [Core Units](https://mips.makerdao.com/mips/details/MIP38#mip38c2-core-unit-state).
```
Comment (blimpa) [inline governance/polls/Ratification Poll for MIP Amendment Subproposals - MIP102c2-SP7 - May 08, 2023.md:55]: Remove?

File: governance/polls/Ratification Poll for MIP Amendment Subproposals - MIP102c2-SP7 - May 08, 2023.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
+---
+title: Ratification Poll for MIP Amendment Subproposals (MIP102c2-SP7) - May 08, 2023
```
Comment (blimpa) [inline governance/polls/Ratification Poll for MIP Amendment Subproposals - MIP102c2-SP7 - May 08, 2023.md:2]:
```suggestion
title: Ratification Poll for MIP Amendment Subproposals (MIP102c2-SP7) - May 8, 2023
```

File: governance/polls/Ratification Poll for MIP Amendment Subproposals - MIP102c2-SP7 - May 08, 2023.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2023-05-08T16:00:00
+end_date: 2023-05-22T16:00:00
+---
+# Ratification Poll for MIP Amendment Subproposals (MIP102c2-SP7) - May 08, 2023
```
Comment (blimpa) [inline governance/polls/Ratification Poll for MIP Amendment Subproposals - MIP102c2-SP7 - May 08, 2023.md:27]:
```suggestion
# Ratification Poll for MIP Amendment Subproposals (MIP102c2-SP7) - May 8, 2023
```

File: governance/polls/Ratification Poll for MIP Amendment Subproposals - MIP102c2-SP7 - May 08, 2023.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+
+The Arbitration Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for fourteen days beginning on Monday, May 08 at 16:00 UTC.
```
Comment (blimpa) [inline governance/polls/Ratification Poll for MIP Amendment Subproposals - MIP102c2-SP7 - May 08, 2023.md:29]:
```suggestion
The Arbitration Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for fourteen days beginning on Monday, May 8 at 16:00 UTC.
```

File: governance/polls/Ratification Poll for MIP Amendment Subproposals - MIP102c2-SP7 - May 08, 2023.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+## Review
+
+The community may vote in this poll to express support or opposition to MIP102c2-SP7 being accepted and implemented in the Maker Protocol.
+
+A summary of this proposal is provided below:
+
+*This subproposal amends the Maker Constitution (now Atlas) and the Scope Frameworks. It aggregates the existing 12 Scope Frameworks into 5: Stability, Support, Protocol, Accessibility, and Governance. Also included are changes to wording and clarification of intent across each amended MIP.*  
+
+Please review the links below to inform your position on this proposal before voting.
+* [Canonical Proposal Version](https://github.com/makerdao/mips/blob/e22e5ed2be970e1b285553028db7eed123531803/MIP102/MIP102c2-Subproposals/MIP102c2-SP7.md)
```
Comment (blimpa) [inline governance/polls/Ratification Poll for MIP Amendment Subproposals - MIP102c2-SP7 - May 08, 2023.md:45]:
```suggestion
* [Canonical Proposal Version](https://github.com/makerdao/mips/blob/5d37617bd4b3cd7cbd714308ff91bdb5564d3002/MIP102/MIP102c2-Subproposals/MIP102c2-SP7.md)
```

**Review by blimpa — CHANGES_REQUESTED** [review]

**Comment by LongForWisdom** [comment]
Appreciate the comments, @blimpa !

File: governance/polls/Amend Keeper Networks - May 8, 2023.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following changes:
+
+
+### Gelato Network Parameters
+
```
Comment (LongForWisdom) [inline governance/polls/Amend Keeper Networks - May 8, 2023.md:33]: Needs finishing, and addresses chasing down.

**Pull request merged**